### ISSUE TITLE
[Quest/Malawi Core] Add finish visit functionality

### ISSUE THREAD

**Issue by KhumboLihonga** [opensrp/fhircore#1585]
**Is your feature request related to a problem? Please describe.**
Currently, there is no way to systematically conclude a visit.

**Describe the solution you'd like**
There needs to be a button at the bottom of the list of tasks in the patient profile. The button will have the label "Finish Visit". The button will be rooted to the bottom and scrolling the tasks will not affect it's visibility. The button will only visible for patients who have a care plan and tasks associated with them i.e. newly diagnosed clients, clients already on ART and exposed infants.

By default, the finish visit button will be greyed out i.e. disabled. This will be the case when none of the tasks have been completed. Once all the visit tasks have been completed, the button will now be blue i.e. enabled. On clicking the enabled button, this should open a questionnaire with the id "patient-finish-visit".

Furthermore, there are two types of visits - a client visit and a guardian visit. For a client visit, all tasks need to be completed before the button is enabled. However, for a guardian visit, only those tasks specified for a guardian visit need to be completed before the button is enabled. These guardian visit tasks are specified according to this [issue](https://github.com/opensrp/fhircore/issues/1470).

**Describe alternatives you've considered**
We tried adding another task called "Finish Visit" but effectively, the user would be able to finish a visit without needing to complete the other tasks.

**Additional context**
Find screenshots below:
![Slide2](https://user-images.githubusercontent.com/90775548/187610889-29cc6cad-6be5-4a4d-b580-499271c70563.PNG)
